VideoLocker/build.gradle

```diff
@@ -122,9 +122,9 @@ dependencies {
      testCompile ("org.mockito:mockito-core:1.9.5"){
          exclude group: 'org.hamcrest'
       }
-     testCompile "org.robolectric:robolectric:3.0-rc3"
-     testCompile "org.robolectric:shadows-support-v4:3.0-rc3"
-     testCompile "org.robolectric:shadows-multidex:3.0-rc3"
+     testCompile "org.robolectric:robolectric:3.0"
```

Comment:
We should probably use the 3.1 SNAPSHOT artifact in order to include a few fixes is submitted since the 3.0 release (which will affect some of our tests).